cabal-install/Distribution/Solver/Modular/Explore.hs

```diff
@@ -67,8 +67,6 @@ backjump (EnableBackjumping enableBj) var initial xs =
 
```

Comment:
@fmthoma Actually, the code that I linked to above isn't enough to prevent space leaks.  I had to fix it in https://github.com/haskell/cabal/pull/4122/commits/41cd78641926bfa05f9aa6d8e75e56be0f8ec18c.

`getFirstGoal` doesn't evaluate the conflict map unless there is more than one goal, so it is possible for a thunk to develop even with --count-conflicts.  Using `seq` is more reliable.